api/data/NMBS/vehicleinformation.php

```diff
@@ -160,6 +160,15 @@ private static function getData($html, $lang, $fast, $occupancyArr, $date, $vehi
                     $departureCanceled = true;
                 }
 
+                // Departed
+                $departureImgNode = $node->children[0]->children[0];
```

Comment:
Good catch, will also keep this in mind in future commits/PR's. Checks are added, including an extra check to see if the src attribute exists (In case they would ever replace the image with a div element etc).